server/models/Column.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const columnSchema = new mongoose.Schema(
```

Comment:
The column schema has no relationship to the Board, add the board ID.

Reply:
Added in upcoming PR